tests/results/dp-perf/edge/edge-plus.md

```diff
@@ -43,23 +49,23 @@ Error Set:
 ## Test3: Running coffee query based routing
 
 ```text
-Requests      [total, rate, throughput]         30000, 1000.00, 999.98
-Duration      [total, attack, wait]             30.001s, 30s, 795.267µs
-Latencies     [min, mean, 50, 90, 95, 99, max]  566.893µs, 746.284µs, 731.685µs, 821.144µs, 856.146µs, 949.534µs, 18.228ms
-Bytes In      [total, mean]                     5040000, 168.00
+Requests      [total, rate, throughput]         29999, 1000.00, 999.97
```

Comment:
1 request is missing?

Reply:
Always a chance for the traffic starting a few milliseconds after intended, which I guess could lead to a missed request.

All 200s which is ultimately what we want.